build.sbt

```diff
@@ -175,32 +176,32 @@ lazy val commonWDS = Seq(
   webpackBundlingMode in fastOptJS := BundlingMode.LibraryOnly(),
   webpackBundlingMode in fullOptJS := BundlingMode.Application,
   test := {},
-  emitSourceMaps := false,
+  scalaJSLinkerConfig in (Compile, fastOptJS) ~= { _.withSourceMap(false) },
+  scalaJSLinkerConfig in (Compile, fullOptJS) ~= { _.withSourceMap(false) },
   // NPM libs for development, mostly to let webpack do its magic
   npmDevDependencies in Compile ++= Seq(
     "postcss-loader"                     -> "3.0.0",
-    "autoprefixer"                       -> "9.7.1",
-    "url-loader"                         -> "2.2.0",
-    "file-loader"                        -> "4.2.0",
-    "css-loader"                         -> "3.2.0",
-    "style-loader"                       -> "1.0.0",
-    "less"                               -> "2.7.2",
-    "less-loader"                        -> "4.1.0",
+    "autoprefixer"                       -> "9.7.6",
+    "url-loader"                         -> "4.1.0",
+    "file-loader"                        -> "6.0.0",
+    "css-loader"                         -> "3.5.3",
+    "style-loader"                       -> "1.2.1",
+    "less"                               -> "3.11.1",
+    "less-loader"                        -> "6.1.0",
     "webpack-merge"                      -> "4.2.2",
-    "mini-css-extract-plugin"            -> "0.8.0",
-    "webpack-dev-server-status-bar"      -> "1.1.0",
+    "mini-css-extract-plugin"            -> "0.9.0",
+    "webpack-dev-server-status-bar"      -> "1.1.2",
     "cssnano"                            -> "4.1.10",
-    "uglifyjs-webpack-plugin"            -> "2.2.0",
-    "html-webpack-plugin"                -> "3.2.0",
+    "terser-webpack-plugin"              -> "3.0.1",
+    "html-webpack-plugin"                -> "4.3.0",
     "optimize-css-assets-webpack-plugin" -> "5.0.3",
-    "favicons-webpack-plugin"            -> "0.0.9",
-    "why-did-you-update"                 -> "1.0.6",
+    "favicons-webpack-plugin"            -> "3.0.1",
     "@packtracker/webpack-plugin"        -> "2.2.0"
   ),
   npmDependencies in Compile ++= Seq(
     "react"            -> reactJS,
     "react-dom"        -> reactJS,
-    "semantic-ui-less" -> SUI,
+    "semantic-ui-less" -> SUILess,
     "prop-types"       -> "15.7.2"
```

Comment:
Do we still need on react 16.13?

Reply:
prop-types? I'm assuming we use it because it's required by some dependency. It's independent from react now. A quick google search turned up an example of usage with react 16.13.1, so I would think it's still relevant.

Reply:
👍